### PR TITLE
Quick fix for TypeScriptEditingTest

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* Add methods for selection and clicking on elements with full matching text
* Fix some typos and logic mistakes
@dmytro-ndp @SkorikSergey @Ohrimenko1988

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java [hunk 1]
```diff
@@ -63,6 +63,10 @@ private boolean isReferenceContainsText(WebElement reference, String expectedTex
     return seleniumWebDriverHelper.waitVisibilityAndGetText(reference).contains(expectedText);
   }
 
+  private boolean isReferenceEqulasText(WebElement reference, String expectedText) {
```
Comment (SkorikSergey) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java]: isReferenceEqulasText  ->  isReferenceEqualsText

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java [hunk 2]
```diff
@@ -63,6 +63,10 @@ private boolean isReferenceContainsText(WebElement reference, String expectedTex
     return seleniumWebDriverHelper.waitVisibilityAndGetText(reference).contains(expectedText);
   }
 
+  private boolean isReferenceEqualsText(WebElement reference, String expectedText) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java]: `isReferenceEqualsText` > `isReferenceEqualsTo`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java [hunk 3]
```diff
@@ -63,6 +63,10 @@ private boolean isReferenceContainsText(WebElement reference, String expectedTex
...
+    return seleniumWebDriverHelper.waitVisibilityAndGetText(reference).equals(expectedText);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java]: We have dedicated command for this case:
```
 seleniumWebDriverHelper.waitTextEqualsTo(reference, expectedText)
```

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java [hunk 4]
```diff
@@ -82,7 +86,17 @@ public void clickOnReference(String visibleText) {
         return;
       }
     }
+    String errorMessage = format("No reference with text \"%s\" has been detected", visibleText);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java]: IMHO next method signature will be more relevant to what method actually does:
```
clickOnReference(String referenceText)
```

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java [hunk 5]
```diff
@@ -82,7 +86,17 @@ public void clickOnReference(String visibleText) {
...
+    throw new RuntimeException(errorMessage);
+  }
 
+  public void clickOnReferenceWithTextEqualsTo(String visibleText) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java]: IMHO next method signature will be more relevant to what method actually does:
```
clickOnReferenceEqualsTo(String referenceText) 
```

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java [hunk 6]
```diff
@@ -93,6 +107,12 @@ public void doubleClickOnReference(String visibleText) {
     seleniumWebDriverHelper.doubleClick();
   }
 
+  public void doubleClickOnReferenceWithTextEqualsTo(String visibleText) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java]: IMHO next method signature will be more relevant to what the method is actually does:
```
doubleClickOnReferenceEqualsTo(String referenceText)
```

**Review by Ohrimenko1988 — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Comment by musienko-maxim** [comment]
ci-build

**Comment by musienko-maxim** [comment]
ci-build

**Pull request merged**